bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/discovery/ShellyThingCreator.java

```diff
@@ -457,7 +457,7 @@ public class ShellyThingCreator {
             Map.entry(SHELLYDT_PRO1PM, THING_TYPE_SHELLYPRO1PM), //
             Map.entry(SHELLYDT_PRO1PM_2, THING_TYPE_SHELLYPRO1PM),
             Map.entry(SHELLYDT_PRO1PM_3, THING_TYPE_SHELLYPRO1PM), //
-            Map.entry(SHELLYDT_PRO1CB, THING_TYPE_SHELLYPRO1CB),
+            Map.entry(SHELLYDT_PRO1CB, THING_TYPE_SHELLYPRO1CB), //
```

Comment:
To avoid conflicts, since unrelated/not needed for this PR, please also revert this:
```suggestion
            Map.entry(SHELLYDT_PRO1CB, THING_TYPE_SHELLYPRO1CB),
```

Reply:
done